src/main/java/Storage.java

```diff
@@ -0,0 +1,202 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Storage class that handles loading tasks from file and saving tasks in the file.
+ */
+public class Storage {
```

Comment:
Since you're storing Strings you might be able to reduce the 3 different storing methods (for todo, deadline, event) into one method, just check if that String you have contains like the indicator of that particular task